libraries/rush-lib/src/cli/parsing/SelectionParameterSet.ts

```diff
@@ -237,14 +240,23 @@ export class SelectionParameterSet {
       })
     );
 
+    const subspaceProjects: Iterable<RushConfigurationProject> = subspaceName
+      ? await this._selectorParserByScope.get('subspace')!.evaluateSelectorAsync({
+          unscopedSelector: subspaceName,
+          terminal,
+          parameterName: 'subspace'
+        })
+      : [];
+
     const selection: Set<RushConfigurationProject> = Selection.union(
       // Safe command line options
       Selection.expandAllDependencies(
         Selection.union(
           toRaw,
           Selection.directDependenciesOf(toExceptProjects),
           // --from / --from-version-policy
-          Selection.expandAllConsumers(fromProjects)
+          Selection.expandAllConsumers(fromProjects),
+          subspaceProjects
```

Comment:
So the contract here is that the `--subspace x` parameter is a shortcut for `--to subspace:x`?

Reply:
No, the `--subspace x` parameter installs the dependencies for just the subspace x

Reply:
I ask because the projects selected by `subspace:x` are currently being injected at the same scope as `--to x` (observe where `toRaw` is placed).

If `--subspace x` is to be treated as `--only subspace:x`, then the value should enter into the overall `selection` expression as a sibling parameter to `onlyProjects`.